modules/ui/sections/raw_tag_editor.js

```diff
@@ -160,32 +156,6 @@ export function uiSectionRawTagEditor(id, context) {
             .merge(list);
 
 
-        // Container for the Add button
```

Comment:
> replaces + button and tab logic

The blank row is an elegant approach, but will removing this button make the remaining ➕ button in the relations section less intuitive? I think there’s probably already some initial confusion over whether clicking it creates a new relation or adds to an existing relation. The analogue to this change would be to always show an empty relation membership row.

Reply:
Good idea, definitely worth experimenting with.